Assets/Scripts/PlayerController.cs

```diff
@@ -26,35 +26,38 @@ public class PlayerController : MonoBehaviour
     [SerializeField] int shotDamage;
     [SerializeField] float shotRate;
     [SerializeField] int shotDist;
-    //////[SerializeField] GameObject recticle;
```

Comment:
The reticle will be an image implemented the same way as the lesson videos! 

Also I added this comment to look cool